server/controllers/projects.js

```
@@ -50,7 +84,72 @@ const createProject = async function (req, res) {
   }
 };
 
+const editProject = async function (req, res) {
+  try {
+    const { projectUsers } = req.body;
+    const updatedProject = await db.project.update(
+      {
+        name: req.body.name,
+        description: req.body.description,
+        createdAt: req.body.createdAt,
+        updatedAt: new Date(),
+      },
+      {
+        where: {
+          id: req.body.id,
+        },
+      }
+    );
+    // If any users have been invited to this project
+    if (projectUsers.length > 0) {
+      for (let i = 0; i < projectUsers.length; i++) {
+        let user = null;
+        if (typeof projectUsers[i] === 'number') {
+          //user already exists in db
+          user = await db.user.findOne({
+            where: {
+              id: projectUsers[i],
+            },
+          });
+        } else if (typeof projectUsers[i] === 'string') {
+          //user doesn't exist in db yet
+          user = await db.user.create({
+            email: projectUsers[i],
+            createdAt: new Date(),
+            updatedAt: new Date(),
+          });
+        } else {
+          //TODO throw error
```

Would you want to throw an error here?